Vagrantfile

```diff
@@ -0,0 +1,15 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# This file loads a generic Vagrantfile that will build the environment
+# specified in the .qi.yml file.
+#
+# DO NOT MODIFY THIS FILE
+#
+# if you want to change the VMs specifications go to README.md
+
+VAGRANT_VMENV_PATH = `node -e "require('vagrant-vmenv')"`.delete!("\n")
```

Comment:
I agree with the concerns, and have created a ticket against the QI project, where this needs to be addressed:

https://issues.gpii.net/browse/GPII-2170